src/mlpack/methods/ann/layer/layer_norm.hpp

```diff
@@ -1,7 +1,8 @@
 /**
  * @file methods/ann/layer/layer_norm.hpp
  * @author Shikhar Jaiswal
- *
+ * * @author Adam Kropp
+
```

Comment:
```suggestion
 * @author Adam Kropp
```
Oops, too many asterisks :)